### PR TITLE
Add shape classes to PojoMapper

### PR DESCRIPTION
**Pull request by dominikl**
While trying to reproduce [Ticket 13009](http://trac.openmicroscopy.org/ome/ticket/13009) Insight crashed, because PojoMapper didn't recognize the shape classes. This PR adds the various shape classes to the PojoMapper.

Test: Open an image, draw an ROI, save ROI. Close image and reopen it again, open Measurement tool again. No error dialog should pop up.

### CODE REVIEW

**Comment by sbesson** [comment]
Closing/reopening to include #4196 in the merge commit for Travis.

**Comment by mtbc** [comment]
What about `Label` and `Path` shapes?

File: components/blitz/src/omero/gateway/util/PojoMapper.java [hunk 1]
```diff
@@ -517,6 +530,20 @@ else if (FilesetData.class.equals(pojoType))
...
+        else if (EllipseData.class.equals(pojoType))
+            return Ellipse.class;
+        else if (LineData.class.equals(pojoType))
+            return Line.class;
+        else if (MaskData.class.equals(pojoType))
+            return Mask.class;
+        else if (PointData.class.equals(pojoType))
+            return Point.class;
+        else if (PolygonData.class.equals(pojoType))
+            return Line.class;
```
Comment (mtbc) [inline components/blitz/src/omero/gateway/util/PojoMapper.java]: not polygon data?

**Comment by mtbc** [comment]
Functional testing went fine.

**Comment by dominikl** [comment]
Thanks @mtbc , once again I missed a copy/paste error; also added Label ROI, but there is no Pojo for Path, so I assume it's not used in Insight.

**Comment by gusferguson** [comment]
Tested using eel user-1 read-only-1.
No error encountered on testing as described.
All ROI types tested.
Did encounter separate issue with text ROIs but checking back to version 5.1.3 this was present then - so will log it as a new bug ticket.
Good to merge.

**Comment by dominikl** [comment]
--no-rebase

**Pull request merged**